doc/source/user_guide/region-processing.rst

```diff
@@ -0,0 +1,42 @@
+.. _region-processing:
+
+Region processing
+=================
+
+Partial region aggregation
```

Comment:
Why is this called "partial" region aggregation? It is a fully functional region aggregation according to three rules.